source/stylesheets/_tab-workflows.scss.erb

```diff
@@ -60,11 +60,14 @@ section.manage-workflows {
 			$li_padding: 12px;
 			$li_fontsize: 16px;
 
-			@include position(absolute, 100%, 0);
+			@include position(absolute, 100%, null, null, 0);
 			z-index: $zindex-workflow_selector;
-			width: 100%;
 			margin-top: 10px;
 			margin-bottom: 0;
+			width: 548px;
+			@include media-below(580px) {
```

Comment:
can you use a variable instead `580px`? I think we have some constants about this

Reply:
There is no 580px constant yet, but most likely it can be calculated somehow based on the `manage-workflows` section since, in the case of 579px, all the buttons except the `workflow-selector` are wrapped into a new line.